config/locales/en.yml

```diff
@@ -16,11 +16,18 @@
 #
 # This would use the information in config/locales/es.yml.
 #
+# The following keys must be escaped otherwise they will not be retrieved by
+# the default I18n backend:
+#
+# true, false, on, off, yes, no
+#
+# Instead, surround them with single quotes.
+#
+# en:
+#   'true': 'foo'
+#
 # To learn more, please read the Rails Internationalization guide
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  activerecord:
-    attributes:
-      user:
-        is_fallback: Fallback
```

Comment:
Removing the `is_` is a trivial example; one from `umts/incidents` is that I have a column `other_vehicle_owned_by_other_driver` which is shorter to type than its translation 'Is the other driver involved the owner of the vehicle?', which is clearer for a person to read than how it would be humanized by default ('Other vehicle owned by other driver?').